src/org/labkey/test/util/core/webdav/WebDavUploadHelper.java

```diff
@@ -133,6 +133,36 @@ public void uploadFile(@NotNull File file)
         uploadFile(file, "");
     }
 
+    public void deleteFile(@NotNull String file, boolean shouldFail)
```

Comment:
Please avoid this pattern of having a flag to specify whether failure is expected. Something like `deleteExpectingError` is preferred.